src/components/address-delivery/address-delivery.js

```diff
@@ -36,37 +36,56 @@ export default class AddressDelivery extends Component {
 
 	async handleSelectStreet(e) {
 		const streetsList = await this.pizzaServiceMock.getStreets();
```

Comment:
DONT DO network requests from Components
Also you shouldn't  create a new instance of pizzaServiceMock = new PizzaServiceMock(); in 9 line